src/axios/index.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios"
+
+const _baseUrl = 'https://backend/uploadImages';
+
+
+export const uploadImagesInstance = axios.create({
+    baseURL: _baseUrl,
+    timeout: 5000,
```

Comment:
5 seconds will not be enough to upload images in real life

Reply:
Will 2 minutes be enough?